portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -26,9 +29,28 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+  /**
+  * Holds the stubbed comments to be returned by the GET method.
+  */
+  private final List<String> comments = Arrays.asList("Comment One.", "Comment Two.", "Comment Three.");
+
+  /**
+  * Utility function that uses Gson to convert a List
+  * (e.g. ArrayList) to a JSON string.
+  */
+  private String listToJson(List list) {
+    Gson gson = new Gson();
+    return gson.toJson(list);
+  }
+
+  /**
+  * Response to a GET request with a JSON string representing the 
+  * hardcoded comments.
+  */
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("Hello Austin!");
+    String json = listToJson(comments);
+    response.setContentType("application/json;");
+    response.getWriter().println(json);
   }
```

Comment:
I love how clean, simple, and concise the `doGet` method ended up in the final version!